### DISCUSSION TITLE
实验报告如何放截图

### DISCUSSION THREAD

**Discussion by Aurora-1412** [arcsysu/SYsU-lang#13] [Q&A]
如题，Debian似乎不支持复制图片

**Comment by wu-kan** [comment]
![A241f739b6003af3c0364e98332ac65c1238b6c9 (1)](https://user-images.githubusercontent.com/44857941/156956489-2783e3c2-4c35-4b77-9670-c395f4ffe227.jpg)

**Reply by Aurora-1412** [reply]
我是不知道如何把截图放到pdf里……就我截图之后不知道如何把图片保存在Debian上，写.md的话也放不进去图

**Reply by wu-kan** [reply]
用word写，然后打印出PDF呢

**Reply by Aurora-1412** [reply]
也就是在windows上写完然后发送到虚拟机上打包提交吗2333

**Reply by wu-kan** [reply]
可以啊
